registrations/tests.py

```diff
@@ -2810,7 +2810,7 @@ def test_registration_process_pmtct_good(self):
         # . check number of calls made:
         #   messageset, schedule, identity, patch identity, jembi registration
         #   identity, reverse identity, reverse identity, patch identity
-        self.assertEqual(len(responses.calls), 9)
```

Comment:
This is a bit weird, we didn't remove any calls did we?

Let investigate by looking at each call and updating the comment above this line.

Reply:
we did remove the try and except, which looked to have (3 default retries)

Reply:
The whole try and except block was only doing one post, which was mocked so it should never have executed the except parts.